src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -174,45 +151,33 @@ export const getLogsURL = (
     nodeName: string,
     Host: string,
     container: string,
+    prevContainerLogs: boolean,
     isResourceBrowserView?: boolean,
     clusterId?: number,
     namespace?: string,
 ) => {
     //const cn = ad.resourceTree.nodes.filter((node) => node.name === nodeName)[0];
-    let prefix = ''
-    if (process.env.NODE_ENV === 'production') {
-        prefix = `${location.protocol}//${location.host}` // eslint-disable-line
-    } else {
-        prefix = `${location.protocol}//${location.host}` // eslint-disable-line
-    }
+    let prefix = `${window.location.protocol}//${window.location.host}` 
```

Comment:
we can directly use the value no need to create variable

Reply:
replaced and removed the the unnecessary variable